failing functional tests on latest: "force customize sync checkbox to be checked", " show password ended with mouseup"

Started with https://tc-test.dev.lcip.org/viewLog.html?buildId=46379&tab=buildResultsDiv&buildTypeId=FxaLatest_FunctionalTests

### force customize sync checkbox to be checked

> AssertionError: expected 'true' to equal 'checked' A true E checked at Command.<anonymous> <tests/functional/lib/helpers.js:1978:24> at Command.<anonymous> <tests/functional/lib/helpers.js:1977:6> at Command.<anonymous> <tests/functional/lib/helpers.js:59:23> at Command.<anonymous> <tests/functional/lib/helpers.js:72:18> at Command.<anonymous> <tests/functional/lib/helpers.js:67:10> at Test.force customize sync checkbox to be checked [as test] <tests/functional/sync_sign_up.js:161:10> at <src/lib/Test.ts:263:51>


### show password ended with mouseup

> AssertionError: expected '' to equal null at Command.<anonymous> <tests/functional/lib/helpers.js:1978:24> at Command.<anonymous> <tests/functional/lib/helpers.js:1977:6> at Command.<anonymous> <tests/functional/lib/helpers.js:59:23> at Command.<anonymous> <tests/functional/lib/helpers.js:72:18> at Command.<anonymous> <tests/functional/lib/helpers.js:67:10> at Test.show password ended with mouseup [as test] <tests/functional/password_visibility.js:46:10> at <src/lib/Test.ts:263:51>

--- a/tests/functional/password_visibility.js
+++ b/tests/functional/password_visibility.js
@@ -43,7 +43,7 @@ registerSuite('password visibility', {
         .then(mouseup('.show-password-label'))
 
         .then(testAttributeEquals('#password', 'type', 'password'))
-        .then(testAttributeEquals('#password', 'autocomplete', null))
+        .then(testAttributeEquals('#password', 'autocomplete', ''))
 
         // \u0008 is unicode for backspace char. By default `type` clears the
         // element value before typing, we want the character to do so.

--- a/tests/functional/sync_sign_up.js
+++ b/tests/functional/sync_sign_up.js
@@ -158,7 +158,7 @@ registerSuite('Firefox Desktop Sync sign_up', {
       return this.remote
         .then(openPage(url, selectors.SIGNUP.HEADER))
 
-        .then(testAttributeEquals(selectors.SIGNUP.CUSTOMIZE_SYNC_CHECKBOX, 'checked', 'checked'));
+        .then(testAttributeEquals(selectors.SIGNUP.CUSTOMIZE_SYNC_CHECKBOX, 'checked', 'true'));
     }
   }
 });

